Upgrade RH SSO to 7.4 (#116)
quarkus-qe/quarkus-openshift-test-suite: README.md

### Enabling/disabling tests

> The `@OnlyIfConfigured` annotation can be used to selectively enable/disable execution of tests based on a configuration property. On the other hand, the `@OnlyIfNotConfigured` annotation will do the same whether the configuration property does not exist.

Reviewer: ```suggestion
The `@OnlyIfConfigured` annotation can be used to selectively enable/disable execution of tests based on a configuration property.
Similarly, the `@OnlyIfNotConfigured` annotation can be used to selectively enable/disable the execution of tests based on a configuration property unavailability.
```
